packages/mui-material/src/transitions/transition.ts

```diff
@@ -28,5 +28,6 @@ export type TransitionKeys =
 export interface TransitionProps
   extends TransitionActions,
     Partial<Pick<_TransitionProps & EasingProps, TransitionKeys>> {
+  id?: string;
```

Comment:
As far as I could see in the issue, is more about being able to pass any valid HTML attribute, not just `id`.

Reply:
👍  agreed